doc/Command-Reference.md

```diff
@@ -1730,6 +1734,87 @@ This command is used to remove particular IPv4 or IPv6 BGP neighbor configuratio
 
 Go Back To [Beginning of the document](#) or [Beginning of this section](#bgp)
 
+## Container Auto-restart
+SONiC includes a feature in which Docker containers can be automatically shut
+down and restarted if one of critical processes running in the container exits
+unexpectedly. Restarting the entire container ensures that configureation is 
+reloaded and all processes in the container get restarted, thus increasing the
+likelihood of entering a healthy state.
+
+### Container Auto-restart show commands
+
+**show container feature autorestart**
+
+This command will display the status of auto-restart feature for containers.
+
+- Usage:
+  ```
+  show container feature autorestart [<container_name>]
+  ```
+
+- Example:
+  ```
+  admin@sonic:~$ show container
+  ```
+
+  ```
+  admin@sonic:~$ show container feature
+  ```
+
+  ```
+  admin@sonic:~$ show container feature autorestart
+  Container Name    Status
+  --------------    --------
+  database          enabled
+  syncd             enabled
+  teamd             disabled
+  dhcp_relay        enabled
+  lldp              enabled
+  pmon              enabled
+  bgp               enabled
+  swss              disabled
+  telemetry         enabled
+  sflow             enabled
+  snmp              enabled
+  radv              disabled
+  ```
+
+Optionally, you can specify a container name in order to display the auto-restart
+feature status for that container only.
+
+### Container Auto-restart config command
+
+**config container feature autorestart <container_name> <autorestart_status>**
+
+This command will configure the status of auto-restart feature for a specific container.
+
+- Usage:
+  ```
+  sudo config container feature autorestart <container_name> (enabled | disabled)
+  ```
+
+- Example:
+  ```
+  admin@sonic:~$ sudo config container
+  ```
+
+  ```
+  admin@sonic:~$ sudo config container feature
+  ```
+
+  ```
+  admin@sonic:~$ sudo config container feature autorestart
+  ```
+
+ ```
+  admin@sonic:~$ sudo config container feature autorestart database
+  ```
```

Comment:
Unnecessary, as these are not complete commands. Please remove.

Reply:
Removed.